src/elements/file-selector/file-selector.ts

```diff
@@ -61,22 +67,22 @@ export class SbbFileSelectorElement extends SbbDisabledMixin(LitElement) {
   /** This will be forwarded as aria-label to the native input element. */
   @property({ attribute: 'accessibility-label' }) public accessibilityLabel: string | undefined;
 
-  /** The list of selected files. */
-  @state() private _files?: File[];
+  /**
+   * The list of selected files.
+   */
+  public get files(): File[] {
+    return this.value || [];
+  }
+  private set _files(value: File[]) {
```

Comment:
I guess it slightly improves readability since we use the getter a lot. 
It seems bad to read from `files` and setting `value` (the main truth is that it was the easiest way to avoid refactoring all the occurrences)

Reply:
Another fair question would be, should we keep the `files` property public?
We could deprecate it in favor of `value` (and just use value everywhere in the code)

Reply:
Copy the native behavior 
- Add a setter for files, the getter returns hiddenInput.files
- The value returned hiddenInput.value (which is the fake path)

Reply:
@jeripeierSBB @kyubisation I know we said that we should comply with the native API where possible but I kept the type of the `files` property to `File[]` instead of `FileList`. FileList is read-only and does not have an initializer (would make the component inconvenient for consumers to use)